.github/workflows/code-review.yml

```diff
@@ -40,13 +41,15 @@ jobs:
 
       - name: Prepare SonarCloud analysis configuration
         run: >
-          mvn -Bclean verify
+          src/oneid/mvnw -Bclean verify
```

Comment:
Could this path be parameterized to reduce replication and avoid errors in case of renaming?